jobs/base_model_validator_errors.py

```diff
@@ -58,10 +61,12 @@ class ModelTimestampRelationshipError(ModelValidationError):
     """Error class for time field model validation errors."""
 
     def __init__(self, model):
+        super(ModelTimestampRelationshipError, self).__init__(model)
         self._message = (
-            'Entity ID %s: The created_on field has a value %s which '
-            'is greater than the value %s of last_updated field'
-            % (model.id, model.created_on, model.last_updated))
+            super(ModelTimestampRelationshipError, self).message + (
```

Comment:
We typically use interpolation in this codebase. Can you do:

`'%s%s' % (..., ...)`

here and below (3 more places)?

Reply:
Also I think you're changing the definition of message here. In the superclass I suggest you define self._base_message or self._message_prefix instead.